yarn-project/simulator/src/acvm/oracle/oracle.ts

```diff
@@ -423,9 +423,4 @@ export class Oracle {
       AztecAddress.fromString(recipient),
     );
   }
-
```

Comment:
This doesn't have to be exposed as an oracle anymore, since we're not requesting logs from `aztec.nr`, just asking for a sync.

Reply:
We will need the oracle however in the future once the sync is done by aztec-nr. Not sure what the best strategy is until then.

Reply:
It is isolated in a private method now. Will add a comment to clarify, but we have enough oracles as it is, I wouldn't expose an unused one.